src/Lykke.Service.BitfinexAdapter.Services/ExecutionHarvester/BitfinexExecutionHarvester.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Autofac;
+using Common;
+using Common.Log;
+using Lykke.Service.BitfinexAdapter.Core.Domain.Trading;
+using Lykke.Service.BitfinexAdapter.Core.Domain.WebSocketClient;
+using Lykke.Service.BitfinexAdapter.Core.Handlers;
+using Lykke.Service.BitfinexAdapter.Core.Utils;
+using Lykke.Service.BitfinexAdapter.Core.WebSocketClient;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.BitfinexAdapter.Services.ExecutionHarvester
+{
+    public sealed class BitfinexExecutionHarvester : IStartable, IStopable
```

Comment:
Don't use IStartable feature (it's AutoFac's issue, use StartupManager instead)

Reply:
Done. BitfinexExecutionHarvester instantiation and starting is now handled by StartupManager